xdsl/ir.py

```diff
@@ -623,24 +623,6 @@ def create(cls: type[OpT],
                                          attributes, successors, regions)
         return cast(OpT, op)
 
-    @classmethod
-    def build(
```

Comment:
Should probably also move op_def

Reply:
So I'm not sure I understand correctly, but I think this is what I did on top, in #700 ?